dev/io.openliberty.jakarta.servlet.6.0/bnd.bnd

```diff
@@ -0,0 +1,36 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 11
+javac.target: 11
+
+Bundle-SymbolicName: io.openliberty.jakarta.servlet.6.0; singleton:=true
+Bundle-Description: Jakarta Servlet, version 6.0
+
+Export-Package: \
+   	jakarta.servlet;thread-context=true;uses:="jakarta.servlet.annotation,jakarta.servlet.descriptor";version="6.0.0",\
+   	jakarta.servlet.annotation;thread-context=true;uses:="jakarta.servlet";version="6.0.0",\
+   	jakarta.servlet.descriptor;thread-context=true;version="6.0.0",\
+   	jakarta.servlet.http;thread-context=true;uses:="jakarta.servlet";version="6.0.0",\
+   	jakarta.servlet.resources;thread-context=true;version="6.0.0"
+
+Include-Resource: \
+  @${repo;io.openliberty.jakarta.servlet:jakarta.servlet-api;6.0.0.20220119;EXACT}!/META-INF/NOTICE
```

Comment:
servlet.5.0 doesn't have an exact listed or a range.  You should validate that when you have this change in that it isn't putting the 6.0 classes in the jakarta.servlet.5.0 bundle now.  I believe that it will because 5.0.0 in the buildpath is only the minimum and not the maximum.  You can change it to `jakarta.servlet:jakarta.servlet-api;version="[5.0.0,6.0.0)` I believe.